internal/eventstore/repository/event.go

```diff
@@ -54,7 +55,7 @@ type Event struct {
 	//ResourceOwner is the organisation which owns this aggregate
 	// an aggregate can only be managed by one organisation
 	// use the ID of the org
-	ResourceOwner string
+	ResourceOwner sql.NullString
```

Comment:
Shouldnt a nullstring be provided as pointer to de sql psckage?

Reply:
why? the nullstring handles the tristate itself and the only reason for a pointer is the scan method